src/AllowedFilterModesExtension.php

```diff
@@ -30,7 +30,7 @@ public function handle(Operation $operation, RouteInfo $routeInfo)
 
         $values = $helper->inferValues($methodCall, $routeInfo);
 
-        $parameter = new Parameter(config(AllowedFilter::FilterModesQueryParamConfigKey), 'query');
+        $parameter = new Parameter(config(AllowedFilter::FilterModesQueryParamConfigKey)??"filter_mode", 'query');
```

Comment:
just exclude AllowedFilterModesExtension from scramble extensions.
That way you won't get any exceptions for reading non-existent config key. There is no need to add this change

Reply:
Yep. I was missing that. Will update on that.  Thank you.